kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
Ya, kato/mort/oort as individual concepts are deprecated. 

Eventually there will be a migration of all the aws/gce code into their respective clouddriver-aws/gce artifacts.